Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -262,6 +274,36 @@ internal static IEnumerator SetHandState(Vector3 handPos, ArticulatedHandPose.Ge
             }
```

Comment:
Wouldn't it be better to return `WaitForFixedUpdate` here?

Reply:
Yeah, probably a good idea, to make sure the input simulation service sees the changed hand data and updates.

On a related note: I have #5173 which makes it takes away the monopoly on simulated hands from the input simulation service. In a similar vein we could actually create true hand controllers in the tests themselves, rather than changing hand data and then waiting for the service to pick up on the changes. That should hopefully reduce the error cases where "wait for one frame" fixes things magically.